src/OpenTelemetry.Exporter.Prometheus.HttpListener/Internal/PrometheusSerializer.cs

```diff
@@ -396,6 +397,40 @@ public static int WriteTags(byte[] buffer, int cursor, Metric metric, ReadOnlyTa
         return cursor;
     }
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static int WriteTargetInfo(byte[] buffer, int cursor, Resource resource)
+    {
+        if (!resource.Attributes.Any())
```

Comment:
This check should be more effecient than calling `Any`.

```suggestion
        if (resource != Resource.Empty)
```